core/src/main/java/com/microsoft/applicationinsights/internal/config/TelemetryConfigurationFactory.java

```diff
@@ -78,9 +79,18 @@
      */
     public final void initialize(TelemetryConfiguration configuration) {
         try {
-            ApplicationInsightsXmlConfiguration applicationInsights = builder.build(getConfigurationFileName());
+            InternalLogger.INSTANCE.configMode(true);
+
+            String configurationFileName = getConfigurationFileName();
```

Comment:
Actually you're looking for the file path and not for the file name which is know (ApplicationInsights.xml)